crates/anstyle-stream/src/lib.rs

```diff
@@ -1 +1,317 @@
+//! **Auto-adapting [`stdout`] / [`stderr`] streams**
+//!
+//! [`Stream`] always accepts [ANSI escape codes](https://en.wikipedia.org/wiki/ANSI_escape_code),
+//! adapting to the user's terminal's capabilities.
+//!
+//! Benefits
+//! - Allows the caller to not be concerned with the terminal's capabilities
+//! - Semver safe way of passing styled text between crates as ANSI escape codes offer more
+//!   compatibility than most crate APIs.
+//!
+//! # Example
+//!
+//! ```
+//! use anstyle_stream::println;
+//! use owo_colors::OwoColorize as _;
```

Comment:
Isn't this supposed to be a color crate on its own? Why do we need `owo_colors`?

Reply:
Oh, so to get termcolor, we must combine `anstyle_stream` + `anstyle`?